app/app.go

```diff
@@ -0,0 +1,23 @@
+package app
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/s1nuh3/academy-go-q32021/routes"
+)
+
+type App struct {
```

Comment:
All exported stuff should have a description:
```go
// App - description here....
type App struct {
```

Reply:
Resolved :)